dev/tidb-in-kubernetes/deploy/tidb-operator.md

```diff
@@ -59,6 +59,10 @@ TiDB 默认会使用很多文件描述符，工作节点和上面的 Docker 进
 
     设置 `LimitNOFILE` 大于等于 `1048576`。
 
+> **注意：**
+>
+> `LimitNOFILE` 需要显示设置为 `1048576` 或者更大，而不是默认的 `infinity`，由于 `systemd` 的 [bug](https://github.com/systemd/systemd/commit/6385cb31ef443be3e0d6da5ea62a267a49174688#diff-108b33cf1bd0765d116dd401376ca356L1186)，`infinity` 在 `systemd` 某些版本中指的是 `65536`。
+
```

Comment:
Suggest to move this config section to https://pingcap.com/docs-cn/stable/tidb-in-kubernetes/deploy/prerequisites/, what do you think?

Reply:
Good suggestion, but this may be another PR.